### ISSUE TITLE
Availability for MS Windows

### ISSUE THREAD

**Issue by vlcinsky** [grafana/tanka#282]
Currently, there is no pre-compiled Windows binary in downloads. Word Windows is mentioned only once in PR #242 but there is no clear statement if Windows is intended target platform or not.

So my question is: what is current and future support for running tanka on Windows:
- none, out of Grafana scope
- best effort - if someone manages to compile or provide PR fixing it, it may work
- intended: when capacity allows

**Comment by sh0rez** [comment]
I'd say best effort is a good description. 

Tanka does compile fine for Windows, and I think it might work pretty well. That being said, it's not a priority for us, because we expect the very biggest amount of our userbase to use UNIX systems – many Windows users will probably have WSL as well.

IMHO we can add `windows/amd64` to our precompiled binaries, so installation becomes easier.
Fixes for that platform will need to be community provided, as I don't think we have team members actively using / developing Tanka on windows.
